fly/eventstream/render.go

```diff
@@ -37,6 +37,10 @@ func Render(dst io.Writer, src eventstream.EventStream, options RenderOptions) i
 			dstImpl.SetTimestamp(e.Time)
 			fmt.Fprintf(dstImpl, "%s", e.Payload)
 
+		case event.SelectedWorker:
+			dstImpl.SetTimestamp(e.Time)
+			fmt.Fprintf(dstImpl, "\x1b[1mselected worker: %s\n", e.WorkerName)
```

Comment:
Looks like this is missing the ANSI reset?

The following will bold only "selected worker:", not sure if we also want to bold the worker name

```suggestion
			fmt.Fprintf(dstImpl, "\x1b[1mselected worker:\x1b[0m %s\n", e.WorkerName)
```

Reply:
Oh, yes, good catch!